tensorflow/tensorflow.bzl

```diff
@@ -441,9 +440,8 @@ def tf_copts(
         # optimizations for Intel builds using oneDNN if configured
         if_enable_mkl(["-DENABLE_MKL"]) +
         if_mkldnn_openmp(["-DENABLE_ONEDNN_OPENMP"]) +
-        if_onednn_v3(["-DENABLE_ONEDNN_V3"]) +
```

Comment:
Please define `onednn_v3_define` in `tsl/mkl/build_defs.bzl` and use it here, e.g., replace
```
if_onednn_v3(["-DENABLE_ONEDNN_V3"]) +
```
with
```
onednn_v3_define() +
```
Where `onednn_v3_define()` returns
```
def onednn_v3_define():
    return select({
        "@org_tensorflow//tensorflow/tsl/mkl:build_with_mkl_aarch64": [],
        "@org_tensorflow//tensorflow/tsl:linux_x86_64": ["-DENABLE_ONEDNN_V3"],
        "@org_tensorflow//tensorflow/tsl:windows": ["-DENABLE_ONEDNN_V3"],
        "//conditions:default": [],
    })
```
(Please also add documentation for `onednn_v3_define`.)

Reply:
Updated.